test/cluster/dynamic_cluster_remove_test.go

```diff
@@ -0,0 +1,260 @@
+package cluster
+
+import (
+	"github.com/hyperledger-labs/orion-server/pkg/types"
+	"github.com/hyperledger-labs/orion-server/test/setup"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+)
+
+func removeServerFromMembersAndNodes(members []*types.PeerConfig, nodes []*types.NodeConfig, serverID int) ([]*types.PeerConfig, []*types.NodeConfig) {
+	switch serverID {
+	case 0:
+		members = members[1:]
+		nodes = nodes[1:]
+	case 1:
+		members = append(members[:1], members[2:]...)
+		nodes = append(nodes[:1], nodes[2:]...)
+	case 2:
+		members = members[:len(members)-1]
+		nodes = nodes[:len(nodes)-1]
+	}
+	return members, nodes
+}
+
+func removeServer(t *testing.T, leaderServer *setup.Server, c *setup.Cluster, removeIndex int, activeServers []int) int {
```

Comment:
The order of the nodes and members is not necessarily the same as the index in the servers array.
Use the node ID instead.
In addition, it seems to me it makes sense to unify or reuse the `removeServerFromMembersAndNodes` above, no?